src/cve/nodes/cve_checklist_node.py

```diff
@@ -65,7 +65,7 @@ async def _parse_list(text: list[str]) -> list[list[str]]:
     for checklist_num, x in enumerate(text):
         try:
             # Try to cut out verbosity:
```

Comment:
This looks good to me, I think we should take this opportunity to improve the original comment. If the point of this line is to turn a string like `'extra stuff [important stuff]\tignore'` into `'[important stuff]'` then we should maybe change the comment to:

```suggestion
            # Remove any text not enclosed by square brackets
```

Reply:
Done